src/PerfView/OtherSources/Linux/LinuxPerfScriptEventParser.cs

```diff
@@ -262,7 +263,22 @@ internal static string GetFileNameWithoutExtension(string path)
             var end = path.LastIndexOf('.');
             if (end < start)
                 end = path.Length;
-            return path.Substring(start, end - start);
+            var name = path.Substring(start, end - start);
+
+            if (stripNiSuffix)
+            {
+                var suffixStart = name.IndexOf(NISuffix);
```

Comment:
You should not assume that this IndexOf will succeed.  Whenever you deal with paths you should use the 'ignore case' option.   The NISuffix variable I don't think should end in a '.'. because name as this point does not end in a '.'.